speech/snippets/transcribe_streaming_v2_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestTranscribeStreamingV2(t *testing.T) {
+	testutil.SystemTest(t)
```

Comment:
the `testutil` library should be used to get the relevant Project ID. 
```suggestion
	tc := testutil.SystemTest(t)
	projectID := tc.ProjectID
```

Reply:
thanks for the suggestions, applied the changes.